README.md

```diff
@@ -1,14 +1,11 @@
 # travis-pyenv
-Set up [pyenv](https://github.com/yyuu/pyenv) to use in [Travis CI](https://travis-ci.org) builds.
-
-Setting up pyenv properly in a Travis CI build environment can be quite tricky. This repo contains a script ([`setup-pyenv.sh`](setup-pyenv.sh)) that makes this process much simpler.
-
-A common use case for this is to install an up-to-date version of [PyPy](http://pypy.org). The Travis CI build images currently contain a very old version of PyPy which breaks some common Python modules.
+With [Travis CI](https://travis-ci.org), sometimes you want to test your project code using an unsupported Python build, such as an exact version of [CPython](http://www.python.org), a more recent version of [PyPy](http://pypy.org), or any number of other Python implementations. You can use [pyenv](https://github.com/yyuu/pyenv) to install a very specific Python version or distribution, but setting it up properly in a Travis CI build environment can be tricky. This repo contains a script ([`setup-pyenv.sh`](setup-pyenv.sh)) you can download and call in your `.travis.yml` configuration to simplify this process.
 
 ## Usage
-1. Set the `$PYENV_VERSION` environment variable to the Python to install.
-2. Tell Travis to cache the `$HOME/.pyenv_cache` directory.
+1. Instead of setting a value for `python` in your `.travis.yml` build matrix, use the `$PYENV_VERSION` environment variable to set the pyenv Python version you want to use for a given build.
```

Comment:
Yes, great point! I think I didn't quite word that right. What I meant was that you can set either `python` or `$PYENV_VERSION` for any individual row/job in the build matrix, but you don't have to have both set for the same job. I.e., this:

```yml
language: python
matrix:
  include:
    - env: PYENV_VERSION='2.7.13' PYENV_VERSION_STRING='Python 2.7.13'
    - python: '3.5'
    - env: PYENV_VERSION=pypy-5.4.1 PYENV_VERSION_STRING='PyPy 5.4.1'
```
Instead of this:

```yml
language: python
matrix:
  include:
    - python: '2.7'
      env: PYENV_VERSION='2.7.13' PYENV_VERSION_STRING='Python 2.7.13'
    - python: '3.5'
    - python: 'pypy'
      env: PYENV_VERSION=pypy-5.4.1 PYENV_VERSION_STRING='PyPy 5.4.1'
```

Both versions have the same result, but the `python` values for jobs where you also have PYENV_VERSION specified I don't think are necessary (and personally I think it clutters things up a little). Without the `python` value, the system Python I think defaults to 2.7, but it doesn't actually affect the pyenv environment.

Anyway, I'll reword that. For my use case in fact I was planning to use pyenv for all of the environments, so that may have been what I was thinking when I wrote it.